packages/server-core/email-templates/account/invite-customer-email.pug

```diff
@@ -0,0 +1,14 @@
+doctype html
```

Comment:
let's rename this to `invite-project-email`

Reply:
So essentially, its not project invite. We are inviting the user to IR engine so that he can signup as customer. And then project owner will add that customer to project.

Reply:
Do we need any of this in the core engine then?

Reply:
Yes, it can be done in MT.

@hurairahmateen FYI